workshops/test/test_event.py

```diff
@@ -158,6 +158,44 @@ def test_delete_event_with_tasks_and_awards(self):
         Task.objects.get(pk=task.pk)
         Award.objects.get(pk=award.pk)
 
+    def test_repository_url(self):
+        test_host = Host.objects.all()[0]
+        links = [
+            'http://user-name.github.com/repo-name',
+            'http://user-name.github.io/repo-name',
+            'https://user-name.github.com/repo-name',
+            'https://user-name.github.io/repo-name',
+            'http://user-name.github.com/repo-name/',
+            'http://user-name.github.io/repo-name/',
+            'https://user-name.github.com/repo-name/',
+            'https://user-name.github.io/repo-name/',
+        ]
+        REPO = 'https://github.com/user-name/repo-name'
+        for index, link in enumerate(links):
+            event = Event.objects.create(
+                slug='e{}'.format(index),
+                host=test_host,
+                url=link
+            )
+            assert event.get_repository_url() == REPO
+
+    def test_website_url(self):
+        test_host = Host.objects.all()[0]
+        links = [
+            'http://github.com/user-name/repo-name',
+            'http://github.com/user-name/repo-name/',
+            'https://github.com/user-name/repo-name',
+            'https://github.com/user-name/repo-name/',
+        ]
```

Comment:
Same question: can you add tests for the case where the URL is completely wrong, and make sure we fail in a useful way?